tests/drivers/eeprom/src/main.c

```diff
@@ -4,17 +4,17 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <zephyr/device.h>
+#include <zephyr/drivers/eeprom.h>
+
 #include <ztest.h>
 #include <ztest_test.h>
```

Comment:
I know this isn't changed here, but would be nice to clean up while we're in here:
1. Including `ztest_test.h` isn't needed, `ztest.h` automatically includes it for us
2. Lets add the `zephyr/` prefix to `ztest.h` since it's now scoped correctly.